.github/workflows/main.yml

```diff
@@ -51,28 +51,25 @@ jobs:
           fi
 
   is-release:
-    # Filtering by `push` events ensures that we only release from the `main` branch, which is a
-    # requirement for our npm publishing environment.
-    # The commit author should always be 'github-actions' for releases created by the
-    # 'create-release-pr' workflow, so we filter by that as well to prevent accidentally
-    # triggering a release.
-    if: github.event_name == 'push' && startsWith(github.event.head_commit.author.name, 'github-actions')
+    name: Determine whether this is a release merge commit
     needs: all-jobs-pass
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
     outputs:
       IS_RELEASE: ${{ steps.is-release.outputs.IS_RELEASE }}
-    runs-on: ubuntu-latest
     steps:
-      - uses: MetaMask/action-is-release@v1
-        id: is-release
+      - id: is-release
+        uses: MetaMask/action-is-release@v2
+        with:
+          commit-starts-with: '[version]'
```

Comment:
I don't think this works, unless you want the commits to be "x.y.z" literally (without "Release" in front).

Reply:
Actually, that's what I wanted, to match the other repos we had like: https://github.com/MetaMask/eth-snap-keyring/commit/a242534fe021a58ba7d244e48d81fe1c9d958f5f

But.. since those commits are already part of the history, this might look weird indeed 🤔

Reply:
Finally I went for `release: [version]`, I took inspiration from angular repo

Reply:
Sounds good. Make sure to document it somewhere to avoid confusion.